src/gui/widgets/CPULoadWidget.cpp

```diff
@@ -91,15 +90,29 @@ void CPULoadWidget::paintEvent( QPaintEvent *  )
 
 void CPULoadWidget::updateCpuLoad()
 {
-	// smooth load-values a bit
-	int new_load = ( m_currentLoad + Engine::audioEngine()->cpuLoad() ) / 2;
-	if( new_load != m_currentLoad )
+	// Additional display smoothing for the main load-value. Stronger averaging
+	// cannot be used directly in the profiler: cpuLoad() must react fast enough
+	// to be useful as overload indicator in AudioEngine::criticalXRuns().
+	const int new_load = (m_currentLoad + Engine::audioEngine()->cpuLoad()) / 2;
+
+	if (new_load != m_currentLoad)
 	{
+		setToolTip(
+			tr("DSP total: ") + QString::number(new_load) + " %\n"
```

Comment:
You can also do like this to avoid unnecessary manual concatenations.
```suggestion
			tr("DSP total: %1").arg(new_load) + " \n"
```